test/unit/index.js

```diff
@@ -6,32 +6,34 @@ require('./api/ws/workers/slaveToMasterSender');
 require('./helpers/ed');
 require('./helpers/jobs-queue');
 require('./helpers/peersManager');
-require('./helpers/pg-notify');
 require('./helpers/request-limiter');
 require('./helpers/RPC');
 require('./helpers/slots');
 require('./helpers/wsApi');
 require('./helpers/z_schema');
 
 require('./logic/account');
-require('./logic/blockReward');
 require('./logic/peer');
```

Comment:
I've missed `logic/multisignature.js`. Do you skip it for a reason?

Reply:
no thanks, it had to be added recently

Reply:
done

Reply:
Please, add as well `schema/delegates` and the extension from `modules/delegates.js`.

Reply:
done